### DISCUSSION TITLE
在Zotero中，在word中已经插入了文献，我怎么从word查看引用的文献在哪对应？

### DISCUSSION THREAD

**Discussion by lsqzero** [zotero-chinese/wiki#302] [求助 / Q&A] [answered]
我插入文献生成列表后，发现文献[1]和文献[3]均在文中对应，但是文献[2]在文中找不到，但我也没法把文献[2]从文档列表中去除。所以我想找一下怎么对应，再删掉对应的引用。

**Comment by zepinglee** [comment]
应该没有办法寻找，尝试搜索一下 `[2]`？

也可能是哪里有 bug，建议：

1. 点击 Refresh；
2. 点击 Document Preferences，切换为其他样式，再切换为原样式，相当于强制刷新。

**Comment by northword** [comment, marked as answer]
感觉是遇到手动编辑引注了，https://zotero-chinese.com/user-guide/faqs/word-addon#%E6%82%A8%E5%B7%B2%E7%BB%8F%E4%BF%AE%E6%94%B9%E8%AF%A5%E5%BC%95%E6%96%87-%E6%98%AF%E5%90%A6%E4%BF%9D%E7%95%99%E6%9B%B4%E6%94%B9

选中全文，右击，切换域代码，然后搜索 `"dontUpdate": true,` 看有没有结果，如果有，删除。

**Comment by lsqzero** [comment]
各位，我仔细检查了一下，是一个跟Axmath域冲突的神奇bug。大体是因为这一段是复制自别的word文档，这个引用的域代码被包含在了axmath的域代码中。引用角标不会在word中显示，但是会在参考文献列表中展现。解决的方法是手动切换域代码，既可以搜索对应的abstract，也可以像northword大佬那样搜索"dontUpdate": true。非常感谢

**Comment by DejaVuDarren** [comment]
我也出现了相同的问题，可是给我一通好找。但最终也找不到原因所在，但我发现最简单的处理方法了。
直接删除掉文中出现问题的引文前后的几个引用，然后剪切这一部分文字内容，再只粘贴文字到原来的问题，再把对应的引文重新引用上，就ok了。这样最简单，也不用找什么原因。
找那些域代码稍微还是有点门槛的。
